js/views/modals/Settings/Store.js

```diff
@@ -0,0 +1,325 @@
+          const errMsg = args[0] && args[0].responseJSON &&
+            args[0].responseJSON.reason || '';
+
+          openSimpleMessage(app.polyglot.t('settings.storeTab.status.error'), errMsg);
+
+          statusMessage.update({
+            msg: app.polyglot.t('settings.storeTab.status.fail'),
+            type: 'warning',
+          });
+        })
+        .always(() => {
+          this.$btnSave.removeClass('processing');
+          setTimeout(() => statusMessage.remove(), 3000);
+        });
+    }
+
+    this.render();
+
+    if (!this.profile.validationError && !this.settings.validationError) {
+      this.$btnSave.addClass('processing');
+    }
+
+    const $firstErr = this.$('.errorList:first');
+    if ($firstErr.length) $firstErr[0].scrollIntoViewIfNeeded();
+  }
+
+  get $btnSave() {
+    return this._$btnSave ||
+      (this._$btnSave = this.$('.js-save'));
+  }
+
+  get $modListSelected() {
+    return this._$modListSelected ||
+      (this._$modListSelected = this.$('.js-modListSelected'));
+  }
+
+  get $modListByID() {
+    return this._$modListByID ||
+      (this._$modListByID = this.$('.js-modListByID'));
+  }
+
+  get $modListAvailable() {
+    return this._$modListAvailable ||
+      (this._$modListAvailable = this.$('.js-modListAvailable'));
+  }
+
+  get $browseMods() {
+    return this._$browseMods ||
+      (this._$browseMods = this.$('.js-browseMods'));
+  }
+
+  get $submitModByIDInput() {
+    return this._$submitModByIDInput ||
+      (this._$submitModByIDInput = this.$('.js-submitModByIDInput'));
+  }
+
+  get $submitModByID() {
+    return this._$submitModByID ||
+        (this._$submitModByID = this.$('.js-submitModByID'));
+  }
+
+  get $submitModByIDInputError() {
+    return this._$submitModByIDInputError ||
+        (this._$submitModByIDInputError = this.$('.js-submitModByIDInputError'));
+  }
+
+  get $submitModByIDInputErrorText() {
+    return this._$submitModByIDInputErrorText ||
+        (this._$submitModByIDInputErrorText = this.$submitModByIDInputError.find('.js-errorText'));
+  }
+
+  render() {
+    loadTemplate('modals/settings/store.html', (t) => {
+      this.$el.html(t({
+        originalMods: this.settings.get('storeModerators').length > 0,
+        errors: {
+          ...(this.profile.validationError || {}),
+          ...(this.settings.validationError || {}),
+        },
+        ...this.profile.toJSON(),
+        ...this.settings.toJSON(),
+      }));
+
+      // this.$('#moderationCurrency').select2();
+
+      this.$profileFormFields = this.$('.js-profileField');
+      this._$btnSave = null;
+      this._$modListSelected = null;
+      this._$modListByID = null;
+      this._$modListAvailable = null;
+      this._$browseMods = null;
+      this._$submitModByIDInput = null;
+      this._$submitModByID = null;
+      this._$submitModByIDInputError = null;
+      this._$submitModByIDInputErrorText = null;
+    });
+
+    if (this.currentMods.length) {
```

Comment:
It also seems weird to me that when I click save, the available mod list reverts to empty with a browse button, forcing me to re-browse. I would think we could just leave in place what's there with maybe a refresh button.